EmployeeManagementService/EmployeeManagementService.Infrastructure/Persistence/EmployeeRepository.cs

```diff
@@ -17,5 +15,25 @@ public async Task<List<Employee>> GetAllEmployees()
                 return await context.Employees.Select(e => new Employee() { FirstName  = e.FirstName, LastName = e.LastName, Role = e.Role, Username = e.Username, Active = e.Active }).ToListAsync();
             }
         }
+
+        public async Task CreateEmployee( string firstName, string lastName, string username, byte[] password, string role, bool active)
+        {
+            using (var context = new RofSchedulerContext())
+            {
+                var employee = new Employee()
+                {
+                    FirstName = firstName,
+                    LastName = lastName,
+                    Username = username,
+                    Password = password,
+                    Role = role,
+                    Active = true   

```

Comment:
no this is not how u set default.

what if in future, i want to pass in false. this is hard coded to true.